src/util.h

```diff
@@ -235,6 +243,55 @@ bool SoftSetBoolArg(const std::string& strArg, bool fValue);
 
 // Forces a arg setting, used only in testing
 void ForceSetArg(const std::string& strArg, const std::string& strValue);
+};
```

Comment:
"Introduce an ArgsManager class encapsulating cs_args, mapArgs and mapMultiArgs"

Indentation for all the methods above is missing. Though it's understandable not to add it now to make diff smaller.